activity-monitor/activity-monitor.go

```diff
@@ -0,0 +1,172 @@
+// Copyright 2014 ISRG.  All rights reserved
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package main
+
+import (
+	"github.com/codegangsta/cli"
+	"github.com/letsencrypt/boulder"
+	"github.com/streadway/amqp"
+	"log"
+	"net/url"
+	"os"
+)
+
+const (
+	QueueName        = "Monitor"
+	AmqpExchange     = "boulder"
+	AmqpExchangeType = "topic"
+	AmqpInternal     = false
+	AmqpDurable      = false
```

Comment:
Correct; the exchange and queues will have to be recreated after broker restart if no persistent messages were left in them.
I don't think there's any issue with having these be transient; we should care instead about marking the appropriate messages as persistent. See http://www.rabbitmq.com/tutorials/amqp-concepts.html